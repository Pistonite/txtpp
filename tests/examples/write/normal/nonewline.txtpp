The write directive should simply write its args to the file.
-TXTPP#write hello
-world
 yay
this line will be on the same line as yay because there's no trailing newline in the write directive.
