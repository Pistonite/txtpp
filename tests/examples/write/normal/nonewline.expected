The write directive should simply write its args to the file.
hello
world
yaythis line will be on the same line as yay because there's no trailing newline in the write directive.
